drivers/SmartThings/matter-sensor/fingerprints.yml

```diff
@@ -33,6 +33,12 @@ matterManufacturer:
     vendorId: 0x1021
     productId: 0x0001
     deviceProfileName: motion-contact-battery
+# HEIMAN
```

Comment:
The `deviceLabel` and `deviceProfileName` should not be in quotes based on the precedent set by the other fingerprints